src/components/pages/AdminAddCourse/AdminAddCoursesForm.js

```diff
@@ -1,181 +1,400 @@
-import React, { useState, useEffect } from 'react';
-import axiosWithAuth from '../../../utils/axiosWithAuth';
-import { useOktaAuth } from '@okta/okta-react';
-import { useDispatch } from 'react-redux';
-import { getCurrentUser } from '../../../redux/actions/userActions';
+import React, { useState } from 'react';
 import { connect } from 'react-redux';
-import { addClass } from '../../../redux/actions/adminActions';
+//import moment from 'moment'; you may need this for auto-pop
+import { useDispatch } from 'react-redux';
+import { useOktaAuth } from '@okta/okta-react';
+import { editCourse, addCourse } from '../../../redux/actions/coursesActions';
 
 // Styles
-import '../../../styles/AdminAddCoursesStyles/AdminAddCoursesStyles.less';
-import { Input, Form, Card } from 'antd';
+import '../../../styles/AdminStyles/AdminEditCourseFormStyles.less';
+import {
+  Input,
+  Form,
+  Checkbox,
+  InputNumber,
+  DatePicker,
+  Button,
+  Modal,
+  TimePicker,
+  Select,
+} from 'antd';
 import TextArea from 'antd/lib/input/TextArea';
-import { useHistory } from 'react-router-dom';
-
-let placeHolder = [];
-let array_string = '';
-let program_list = [];
-let formPrerequisite = '';
+const { Option } = Select;
 
 const initialFormValues = {
-  class_name: '',
-  class_subject: '',
-  class_desc: '',
+  course_id: '',
+  course_name: '',
+  course_description: '',
+  days_of_week: [],
+  max_size: '',
+  max_age: '',
+  min_age: '',
+  start_date: '',
+  end_date: '',
+  start_time: '',
+  end_time: '',
+  number_of_sessions: '',
+  instructor_id: '',
+};
+
+// may be necessary for auto-pop
+// const daysOfWeek = [
+//   'Monday',
+//   'Tuesday',
+//   'Wednesday',
+//   'Thursday',
+//   'Friday',
+//   'Saturday',
+//   'Sunday',
+// ];
```

Comment:
This array was originally being used to create the check boxes in the form. It looks like that has been done below now, so this array can be removed.